proarc-common/pom.xml

```diff
@@ -200,6 +200,26 @@
             <artifactId>postgresql</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>cz.cas.lib.proarc</groupId>
+            <artifactId>proarc-aes57</artifactId>
+            <version>3.5-SNAPSHOT</version>
+        </dependency>
+        <dependency>
```

Comment:
Ta dependency je 4 x. To asi přidala Idea, ne?

Reply:
Vyřešeno